rundeckapp/grails-app/views/framework/_pluginConfigPropertyFormField.gsp

```diff
@@ -162,15 +162,16 @@
 
             <g:each in="${propSelectValues}" var="propval" status="n">
                 <div class="optionvaluemulti checkbox">
-                  <g:checkBox name="${fieldname}" checked="${propval.value in defvalset}" value="${propval.value}" id="f_${fieldid}_p_${n}"/>
-                  <label class="grid-row optionvaluemulti" for="f_${fieldid}_p_${n}">
-                    ${propval.label}
-                  </label>
-                </div>
-            </g:each>
-        </div>
-        </div>
-    </g:elseif>
+              <g:set var="ischecked" value="${propval.value in defvalset ? 'checked="checked"' : ''}"/>
+            <input type="checkbox" id="${fieldname}" name="${fieldname}" ${ischecked} value="${propval.value}"/>
```

Comment:
non gsp tags will need attributes to be encoded
```suggestion
            <input type="checkbox" id="${enc(attr:fieldname)}" name="${enc(attr:fieldname)}" ${ischecked} value="${enc(attr:propval.value)}"/>
```

Reply:
If you selected multiple options of the checkbox group they would all be sent to the backend as empty strings. eg ["","",""]. Once I made the change the correct values were transferred.

Reply:
ok, thanks